pages.de/windows/cipher.md

```diff
@@ -0,0 +1,16 @@
+# cipher
+
+> Verschlüsselt oder entschlüsselt Dateien auf NTFS-Laufwerken.
+> Weitere Informationen: <https://learn.microsoft.com/de-de/windows-server/administration/windows-commands/cipher>.
+
+- Verschlüsseln einer Datei oder eines Verzeichnisses:
```

Comment:
Could you please be a bit more descriptive? What does happen with the encrypted file or directory?
```suggestion
- Verschlüssle eine Datei oder ein Verzeichnis:
```

Reply:
I didn't go into more detail because I only translated the original page. Should I update this then also or may the translations partly deviate?

Reply:
Oh, sorry, I didn't think that far. The English page should always be the most up-to-date. (So if you want to make this change, it should also be applied to the main page.)

Reply:
okay, but I will do this in a separate PR.